irony-cdb-json.el

```diff
@@ -100,6 +114,30 @@ it could easily be used with helm, for instance, by enabling
     (irony-cdb-json--save-project-alist)
     (irony-cdb-autosetup-compile-options)))
 
+(defun irony-cdb-json--last-mod (file)
+  (nth 5 (file-attributes file)))
+
+;;;###autoload
+(defun irony-cdb-json-select-most-recent ()
+  "Select CDB that is most recently modified."
+  (interactive)
+  (let (exists notexists)
```

Comment:
FYI `(let (exist notexists))` already initialize the member to `nil`, `nil` and `'()` are equivalent, so you don't need the 2 set lines.

And you would not want to do `let` + `setq`, just initialize in the `let`: `(let ((a '()) (b '()))`